plugins/include/string.inc

```diff
@@ -168,6 +173,23 @@ native format(output[], len, const format[], any:...);
 native formatex(output[], len, const format[], any:...);
 
 /**
+ * Formats and returns a string according to the AMX Mod X format rules
+ * (see documentation).
```

Comment:
A wiki page like SM has but for AMXX. It has yet to be created. Feel free to do it!